test/e2e/framework/skipper/skipper.go

```diff
@@ -133,6 +133,13 @@ func SkipUnlessLocalEphemeralStorageEnabled() {
 	}
 }
 
+// SkipUnlessPodOverheadEnabled skips test if PodOverhead feature gate is not enabled.
```

Comment:
`SkipIfPodoverheadDisabled` feels easier to read to me

Reply:
I was just following the pattern that existed to be consistent with what was already there for feature gates.

Would you rather see both changed?

Reply:
+1

Reply:
addressed.